frontend/packages/knative-plugin/src/models.ts

```diff
@@ -105,7 +105,7 @@ export const EventSourceApiServerModel: K8sKind = {
   kind: 'ApiServerSource',
   label: 'ApiServerSource',
   labelPlural: 'ApiServerSources',
-  plural: 'ApiServerSources',
+  plural: 'apiserversources',
```

Comment:
Are you sure about this change? This feels like it doesn't follow standard API conventions...

eg
```
export const PipelineRunModel: K8sKind = {
  apiGroup: 'tekton.dev',
  apiVersion: 'v1alpha1',
  label: 'Pipeline Run',
  plural: 'pipelineruns', // <<<<<<
  abbr: 'PLR',
  namespaced: true,
  kind: 'PipelineRun',
  id: 'pipelinerun',
  labelPlural: 'Pipeline Runs',
  crd: true,
  badge: BadgeType.DEV,
};
```
`plural` is an API-level name, not a readable name. So it's "camelCase".

Reply:
Hah, apparently I can't read... you are putting it to lowercase, not the other way around.

Reply:
yeah `plural`  is used to fetch resources so all models has lowercase